### PR TITLE
Error on attempting to restart container

### PR DESCRIPTION
**Pull request by mheon**
Until we have Conmon support, we need to inform people this isn't possible

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: libpod/container_api.go [hunk 1]
```diff
@@ -313,6 +313,11 @@ func (c *Container) Start() error {
 		return errors.Wrapf(ErrCtrStateInvalid, "container %s must be in Created or Stopped state to be started", c.ID())
 	}
 
+	// TODO remove this when we patch conmon to support restarting containers
+	if c.state.State == ContainerStateStopped {
+		return errors.Wrapf(ErrNotImplemented, "restarting a stopped container is not yet supported")
```
Comment (TomSweeneyRedHat) [inline libpod/container_api.go]: Do we need a note in the man page too?

Comment (baude) [reply libpod/container_api.go]: i dont think so, just release notes maybe

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**